### DISCUSSION TITLE
How to skip trials that end in a wrong model

### DISCUSSION THREAD

**Discussion by fbettcba** [keras-team/keras-tuner#565] [Q&A] [answered]
I'm wondering if there is a way to skip trials that end up in a combination of hyperparameters that cause the model to fail building. 
For example some combinations end up in errors like: Negative dimension size caused by subtracting 3 from 1 for. 
And the process ends with an error: Too many failed attempts to build model.

Is there any way to just ignore the trials that result in error? and continue searching other options?

**Comment by haifeng-jin** [comment, marked as answer]
Currently, there is no quick solution to this, you may need to make some if statements in the `build` function, when you use the hps to create the model.

**Comment by invernizzi** [comment]
It's not an official API, but you can control the number of failed model builds that will be attempted before Keras Tuner gives up. 
```
import keras_tuner as kt

kt.engine.tuner.MAX_FAIL_STREAK = 60
```
